src/BaseOfTalents/CVParser/Core/DocToStringListConverter.cs

```diff
@@ -69,11 +69,15 @@ private bool TryReadText(string path, out string text, List<FuncParser> textLoad
 
         private static string GetFirstText(string path, List<FuncParser> textLoaders)
         {
-            var errors = new List<Exception>();
-            string text = default(string);
-            int i = 0;
-
-            for (var loader = textLoaders[i]; i < textLoaders.Count && !string.IsNullOrEmpty(text); i++)
+            List<Func<string, string>> defaultTextLoaders = new List<Func<string, string>>()
```

Comment:
we have a functions that do the same thing, remember it
just rename textLoadersto actualLoaders and get out of this initialization